src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -122,7 +122,12 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
             setSelectedDayIndex(dayIndex);
           }}
         >
-          {formatDayLabel(day)}
+          <button
+            aria-label={getWeekday(day)}
```

Comment:
Why not use `formatDayLabel(day)`? It already exists and it transforms the first two days into `Today` and `Tomorrow`

Reply:
Done

Reply:
+1. Remove `getWeekday` it's no longer used.

Reply:
removed